Makefile

```diff
@@ -1,8 +1,9 @@
-OADP_TEST_NAMESPACE ?= oadp-operator
-CREDS_SECRET_REF ?= cloud-credentials
-OADP_AWS_CRED_FILE ?= /var/run/oadp-credentials/aws-credentials
-OADP_S3_BUCKET ?= /var/run/oadp-credentials/velero-bucket-name
-VELERO_INSTANCE_NAME ?= example-velero
+# OADP_TEST_NAMESPACE ?= oadp-operator
+# CREDS_SECRET_REF ?= cloud-credentials
+# OADP_AWS_CRED_FILE ?= /var/run/oadp-credentials/aws-credentials
+# OADP_S3_BUCKET ?= /var/run/oadp-credentials/velero-bucket-name
+# VELERO_INSTANCE_NAME ?= example-velero
```

Comment:
I dont think we are setting these in the CI environment. Is there any specific reason for commenting this?